src/shapes/input_text/InputText.js

```diff
@@ -14,9 +14,25 @@ const options = {
       type: 'dropdown',
       choices: ['Cursive', 'Sans Serif', 'Monospace'],
     },
+    rotateDir: {
+      title: 'Rotate',
+      type: 'dropdown',
+      choices: ['Top', 'Center', 'Bottom'],
```

Comment:
I think these should be lowercase for consistency.

Reply:
This controls the text in the options, so the user sees them. I think Capital makes more sense for that. The fonts are also Capitalized. I know the shape names are lowercase, but they aren't seen in the UI.